### PR TITLE
reordered themes in central Header class to include bootstrap first, …

### PR DESCRIPTION
**Pull request by bradymiller**
…then followed by main theme

This is dedicated to @sjpadgett !
@robertdown , tagging you to make sure I didn't embarrass myself working in your Header class

Can demo this at:
http://www.open-emr.org/wiki/index.php/Development_Demo#Kappa_-_Up_For_Grabs_Demo

### CODE REVIEW

**Comment by bradymiller** [comment]
Demo is on the way...

**Comment by bradymiller** [comment]
demo is at:
Can demo this at:
http://www.open-emr.org/wiki/index.php/Development_Demo#Kappa_-_Up_For_Grabs_Demo

**Comment by sjpadgett** [comment]
lol, Dedicated to or who to contact if it blows up!

**Comment by bradymiller** [comment]
both

File: interface/billing/ub04_form.php [hunk 1]
```diff
@@ -36,8 +36,11 @@
 <meta charset="utf-8" />
 <?php if ($isAuthorized !== true) {?>
 <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-3-1-1/index.js"></script>
- <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
- <link href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-ui-1-12-1/themes/base/jquery-ui.min.css" rel="stylesheet" type="text/css" />
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
+<?php } ?>
```
Comment (bradymiller) [inline interface/billing/ub04_form.php]: This is not a good idea here. Will revert this code block.

File: interface/forms/eye_mag/view.php [hunk 2]
```diff
@@ -221,14 +221,17 @@ function sel_diagnosis(target,term) {
     </script>
 
     <!-- Add Font stuff for the look and feel.  -->
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+    <?php if ($_SESSION['language_direction'] == 'rtl') { ?>
+        <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
+    <?php } ?>
```
Comment (bradymiller) [inline interface/forms/eye_mag/view.php]: Need to test the eye stuff in rtl (if looks funny, then will revert the rtl stuff in this and above eye forms)
TODO

**Comment by bradymiller** [comment]
Just brought in changes to also address this in places that are not using the standard Header class.
(also refreshed the up for grabs demo to use this code)

**Comment by bradymiller** [comment]
@sjpadgett , @robertdown ,
I think this is ready to get into the codebase. The earlier the better, so can identify and fix any bugs (I only found a couple styling issues so far which I have addressed and appears to be working well in all 3 styling families (that is light, manila, and flat themes)). Just let me know if you have any issues with me bringing this into the codebase.

**Comment by sjpadgett** [comment]
Sorry Brady missed this. Thanks for this though because, in the end I believe it's better all the way around.

**Comment by bradymiller** [comment]
Thanks for pointing this issue out. Was much better dealing with this now than in the future when it would of likely been more difficult.

**Pull request merged**